src/components/AgendaItems/Create/AgendaItemsCreateModal.tsx

```
@@ -152,185 +155,195 @@ const AgendaItemsCreateModal: React.FC<
+                      </video>
+                    ) : (
+                      <img src={attachment} alt={t('attachmentPreview')} />
+                    )}
+                    <button
+                      className={styles.closeButtonFile}
+                      onClick={(e) => {
+                        e.preventDefault();
+                        handleRemoveAttachment(attachment);
+                      }}
+                      data-testid="deleteAttachment"
                     >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)